src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/AdmissionControlDecider.java

```diff
@@ -44,18 +51,35 @@ public String name() {
     @Override
     public Decision operate() {
         Decision decision = new Decision(Instant.now().toEpochMilli(), NAME);
-        counter += 1;
-        if (counter < decisionFrequency) {
```

Comment:
Why have we removed the `decisionFrequency` from operate logic ? Do we want to run this decider every 5seconds ?